File: QandAProject/src/app/classes/vote.ts

export class Vote {

  public dateVote: Date;

  public id: number;

  public idAnswer: number;

  public voter: string;

  public isUserVote: boolean;

  public idSurvey: number;

  constructor(idAnswer: number, idSurvey: number) {
    this.dateVote = new Date();
    this.id = 0;
    this.idAnswer = idAnswer;
    this.idSurvey = idSurvey;
  }
}

File: Server/Entyties/Models/Vote.cs
using System;

namespace Entities.Models
{
	public class Vote//голос
	{
		public int Id { get; set; }

		public int IdAnswer { get; set; }//ид ответа за который отдан голос

		public int IdCustomer { get; set; }//ид юзер проголосовавшего за ответ

		public DateTime DateVote { get; set; }//когда голосовал

		public int IdSurvey { get; set; }

	}
}
